examples/responsive-screen-orientation-change/App.js

```diff
@@ -10,11 +10,14 @@ import {
  
 export default class App extends React.Component {
   componentWillMount() {
-    listenOrientationChange(this);
+   // `listenOrientationChange` returns the handler method,
+   // consider store it to be pushed back when removing the listener.
+   this.orientationChangeHandler = listenOrientationChange(this);
   }
 
   componentWillUnMount() {
-    removeOrientationListener();
+    // Optionally path the original handler to be removed.
```

Comment:
no not optionally! It has to be removed, because if the component remounts then you will have 2 or more listeners registered. That leads to memory leaks and can make the app crash